cloudevents/extensions/attrschema.md

```diff
@@ -0,0 +1,156 @@
+# Attrschema Extension
+
+## Attributes
+
+#### attrschema
+
+- Type: `URI`
+- Description: Identifies the [Attrschema](#attrschema-document) that attributes of
+  the current CloudEvent adhere to.
+ 
+  Incompatible changes to the schema SHOULD be reflected by a different URI. See
```

Comment:
Not sure this first sentence is needed since it's not saying what it's incompatible with. Either this URI describes the current attributes or not.